### ISSUE TITLE
The subset field is a foreign key on outputs and not manytomany relations.

### ISSUE THREAD

**Issue by janekg89** [matthiaskoenig/pkdb#669]
It is only possible to use a output for one subset. Should it be possible to use an output for multiple purposes?

**Comment by matthiaskoenig** [comment]
Yes, outputs should be usable in multiple subsets. I.e. I often have timecourse vectors of multiple species and want to create all the scatters between them.
